fe/fe-core/src/main/java/com/starrocks/feature/ProductFeature.java

```diff
@@ -57,6 +57,11 @@ public String getLink() {
                 "privilege system with full RBAC functionalities, supporting role inheritance and default roles.",
                 "https://docs.starrocks.io/en-us/latest/administration/privilege_overview"
         ));
+        features.add(new ProductFeature(
+                "multi-warehouse",
+                "support multiple warehouses, each with its own worker group and configuration.",
```

Comment:
Thanks for this PR 👍 
I have a questions about `its own worker group and configuration`: how can we inject the computer nodes for this warehouse?